arm-storsimple8000series/2017-06-01/swagger/storsimple.json

```diff
@@ -905,6 +905,44 @@
         },
         "deprecated": false
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.StorSimple/operations": {
```

Comment:
I don't think this is correct, the operations should be at the provider level.  See https://github.com/Azure/azure-rest-api-specs/blob/master/arm-redis/2016-04-01/swagger/redis.json#L37 for an example.